airflow/bin/cli.py

```diff
@@ -719,13 +1263,32 @@ class CLIFactory(object):
             ("-kt", "--keytab"), "keytab",
             nargs='?', default=conf.get('kerberos', 'keytab')),
         # run
+        # TODO(aoen): "force" is a poor choice of name here since it implies it overrides
+        # all dependencies (not just past success), e.g. the ignore_depends_on_past
+        # dependency. This flag should be deprecated and renamed to 'ignore_ti_state' and
+        # the "ignore_all_dependencies" command should be called the"force" command
+        # instead.
         'force': Arg(
             ("-f", "--force"),
-            "Force a run regardless or previous success", "store_true"),
+            "Ignore previous task instance state, rerun regardless if task already "
+            "succeeded/failed",
+            "store_true"),
         'raw': Arg(("-r", "--raw"), argparse.SUPPRESS, "store_true"),
+        'ignore_all_dependencies': Arg(
+            ("-A", "--ignore_all_dependencies"),
+            "Ignores all non-critical dependencies, including ignore_ti_state and "
+            "ignore_task_deps"
```

Comment:
I think the difference between `ignore_task_deps` and `ignore_all_dependencies` is a bit confusing. From another doctstring: 
```
        :param ignore_all_deps: Ignore all ignoreable dependencies.
            Overrides the other ignore_* parameters.
        :type ignore_all_deps: boolean
        :param ignore_depends_on_past: Ignore depends_on_past parameter of DAGs
            (e.g. for Backfills)
        :type ignore_depends_on_past: boolean
        :param ignore_task_deps: Ignore task-specific dependencies such as depends_on_past
            and trigger rule
        :type ignore_task_deps: boolean
```
https://github.com/apache/incubator-airflow/blob/1.8.0/airflow/models.py#L877-L885